appinsights/src/channel/mod.rs

```diff
@@ -14,7 +14,7 @@ use crate::contracts::Envelope;
 /// An implementation of [TelemetryChannel](trait.TelemetryChannel.html) is responsible for queueing
 /// and periodically submitting telemetry events.
 #[async_trait]
-pub trait TelemetryChannel {
+pub trait TelemetryChannel: Send + Sync {
```

Comment:
Strictly speaking `Sync` is not required to share client b/w threads. Can we remove this? It might cause issues in future if left like this